lexicons/com/atproto/admin/enableAccountInvites.json

```diff
@@ -11,7 +11,11 @@
           "type": "object",
           "required": ["account"],
           "properties": {
-            "account": {"type": "string", "format": "did"}
+            "account": { "type": "string", "format": "did" },
+            "note": {
```

Comment:
do we need a note on enable? or should this just unset the existing note?

Reply:
In my mind, if someones invites were disabled, we would probably want to explain why we are re-enabling them but not too sure if that adds up with our actual use-case. @bnewbold said we're tracking all of this in an excel right now, do we also track reasons for re-enabling invite codes?